shared/AppInsightsCore/src/JavaScriptSDK/CoreUtils.ts

```diff
@@ -182,33 +182,6 @@ export function perfNow(): number {
     return dateNow();
 }
 
-/**
- * Generate random base64 id string.
- * The default length is 22 which is 132-bits so almost the same as a GUID but as base64 (the previous default was 5)
- * @param maxLength - Optional value to specify the length of the id to be generated, defaults to 22
- */
-export function newId(maxLength = 22): string {
```

Comment:
Moved to RandomHelper, to avoid some circular dependencies that are going to crop up later.